docs/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Deploy TiDB to Kubernetes on Google Cloud
+summary: Tutorial for deploying TiDB on Google Cloud using Kubernetes.
+category: operations
+---
+
+# Deploy TiDB, a distributed MySQL compatible database, to Kubernetes on Google Cloud
+
+## Introduction
+
+This tutorial is designed to be [run in Google Cloud Shell](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator).  It takes you through these steps:
+
+- Launching a new 3 node Kubernetes cluster (optional)
+- Installing the Helm package manager for Kubernetes
+- Deploying the TiDB Operator
+- Deploying your first TiDB cluster
+- Connecting to TiDB
+- Shutting down down the Kubernetes cluster (optional)
+
+## Select a Project
+
+Please select a project before proceeding.  The approximate cost for running the compute resources in this tutorial is 10 cents/hour.
+
+<walkthrough-project-billing-setup key="project-id">
+</walkthrough-project-billing-setup>
+
+## Enable API Acess
+
+This tutorial will require use of the Compute and Container APIs.  Please enable them before proceeding:
+
+<walkthrough-enable-apis apis="container.googleapis.com,compute.googleapis.com">
+</walkthrough-enable-apis>
+
+## Configure gcloud Defaults
+
+This step defaults gcloud to your prefered project and [zone](https://cloud.google.com/compute/docs/regions-zones/), which will simplify the commands used for the rest of this tutorial:
+
+	gcloud config set project {{project-id}} &&
+	gcloud config set compute/zone us-west1-a
+
+## Launch a three node Kubernetes cluster
+
+It's now time to launch a 3 node kubernetes cluster! The following command launches a 3 node cluster of `n1-standard-1` machines.
+
+It will take a few minutes to complete:
+
+	gcloud container clusters create tidb
+
+Once the cluster has launched, set it to be the default:
+
+	gcloud config set container/cluster tidb
+
+The last step is to verify that `kubectl` can connect to the cluster, and all three machines are running:
+
+	kubectl get nodes
+
+If you see `Ready` for all nodes, congratulations!  You've setup your first Kubernetes cluster.
+
+## Installing Helm
+
+Helm is the package manager for Kubernetes, and is what allows us to install all of the distributed components of TiDB in a single step.  Helm requires both a server-side and a client-side component to be installed.
+
+Because Cloud Shell system packages do not persist, we also copy the `helm` command to a local directory.
+
+Download the `helm` installer:
+
+	curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+
+Install `helm`:
+
+	chmod 700 get_helm.sh && ./get_helm.sh
+
+Copy `helm` to a local directory:
+
+	mkdir -p ~/bin &&
+	cp /usr/local/bin/helm ~/bin &&
```

Comment:
Usually /usr/local/bin is already part of `PATH` environment variable. Copying to $HOME/bin is unnecessary.

Reply:
This is explained in this sentence just above:

> Because Cloud Shell system packages do not persist, we also copy the `helm` command to a local directory.
 
On Idle, the Cloud Shell wipes everything except the user's $HOME.

Reply:
Oh, I thought the gcloud is executed on a local machine.

Reply:
You can do that, and you can also install a local Cloud Shell.

But from the web console, Google Cloud has a 'Shell' button (opens in a browser).  All their own tutorials use it, and it's quite convenient.  This PR also adds a big blue button 'Open in Cloud Shell' to `README.md`, which will only work correctly after this PR is pulled (works off master branch).

Reply:
Yes, I know. It's just that I forgot this when reviewing these commands.

Reply:
We can move the `Because...` to where the `Copy` is.

Reply:
@gregwebs +1 makes sense to me.